restful.admin.inc

```diff
@@ -59,6 +59,20 @@ function restful_admin_settings($form_state) {
     ),
   );
 
+  $description  = 'When you make an XHR (AJAX) call to a resource under a ';
+  $description .= 'different host, most modern browsers will make an initial ';
+  $description .= 'OPTIONS request to the resource. The response to that can ';
+  $description .= 'contain the Access-Control-Allow-Origin, if that includes ';
+  $description .= 'the domain making the request the browser will allow the ';
+  $description .= 'cross-domain request. The contents of this field will be ';
+  $description .= 'used in that header.';
+  $form['restful_allowed_origin'] = array(
+    '#type' => 'textfield',
+    '#title' => t('CORS Preflight'),
```

Comment:
> The cors preflight is a (geeky) implementation detail of the allowed origin.

:+1:

> you shouldn't pass variables to t(), only the text itself.

I only put it in a variable to split it in different lines, to help with readability.